docs/to_clarify.md

```diff
@@ -1 +1,23 @@
-1. Why does identification verification require "booking document + passport + credit card authorization"?
+1. Stated under the section "Problem Statement" that the solution must be "safe and secured to protect guests' privacy". What exactly does this requirement mean, for example, are there certain security standards that the solution must adhere to?
+
+1. Stated under the section "Possible Use Cases" that the solution should "enhance back-end operational capabilities and business intelligence" which will be used to "create accurate guest profiles and provide personalized and targeted guest experiences". In what manner is the solution expected to do this?
+
+1. Stated under the section "Possible Use Cases" that the solution should "allow data transference capabilities, from room requests collected on the booking engine, to the check-in process till checkout".
+
+    Does this mean that the "digital mobile check-in and check-out solution" is expected to do more than digitising the checking-in and checking-out processes; it has to integrate with other platforms to provide other additional services, beginning from the booking of hotel rooms till checkout?
+
+1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
+
+    a. What needs to be done during the checking-in process?
+
+    b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
+
+    c. On the same note, what needs to be done during the checking-out process?
+
+1. Stated under the section "What Are We Looking For" that the solution should "verify travel and identification documents, booking documents, payment details, and make payment upon check-out, integrated with ICA (or other government stipulated system) for authentication and verification".
+
+    a. Which government stipulated system is currently being used, and does the existing system provide API for our solution to communicate with it?
+
+    b. Can we have several different samples of the above documents and the details of what are being verified for these documents? // maybe can be integrated with going through check-in & check-out process
+
+1. Stated under the section "What Are We Looking For" that the solution should "streamline the back-end allocation of rooms by ensuring room requests are clearly accounted for on the back-end, to ensure rooms and amenities are available upon guest arrival". How is this currently being done? // maybe can be integrated with going through check-in & check-out process.
```

Comment:
I am not sure if we should assume that they have done this. It might be a new feature that they are looking for. So, maybe the question "How is this currently being done?" can be reworded into something else.

Reply:
I assumed that it's currently being done because it's about ensuring that amenities and the room is ready, and the room requests are being handled properly - which concerns the quality of their service.

But yes, I could first ask whether this is currently being done, and if so, how is it currently being done.